static/js/syntaxModesRules.ts

```diff
@@ -63,266 +84,265 @@ const LEVELS = [
   {
     // Adds lists and 'at random'
     name: 'level2',
-    rules: pipe(baseRules(),
+    rules: pipe(baseRules(currentLang._AT,currentLang._RANDOM),
 
-      rule_printSpace('expression_eol'),
-      rule_isAsk('gobble'),
-      rule_is('gobble'),
-      rule_turtle(),
-      rule_sleep(),
+      rule_printSpace(currentLang._PRINT,'expression_eol'),
+      rule_isAsk(currentLang._IS,currentLang._ASK,'gobble'),
+      rule_is(currentLang._IS,'gobble'),
```

Comment:
I see that this works, but I am not sure it is very pretty with all the extra arguments. Could we add the whole currentLang to the global state and access it in the functions?

Reply:
New version looks a lot better!!